lib/screens/login_screen.dart

```diff
@@ -49,14 +50,15 @@ class _LoginScreenState extends State<LoginScreen> {
     User userProfile = await loadUserProfile(accessToken);
     SharedPreferences prefs = await SharedPreferences.getInstance();
 
-    if(!prefs.containsKey('username')){
+    if(!prefs.containsKey('username')){ // first install
       await prefs.setString('username', userProfile.username);
```

Comment:
what if the preference contains the key but it is empty? is it worth covering for that case?

Reply:
I can't think of a situation where the key exists but the value not, since I never remove the key and/or value.